BWEnv/include/zmq_server.h

```diff
@@ -38,9 +38,9 @@ class ZMQ_server
   void connect();
   void close();
   void sendHandshake(const torchcraft::fbs::HandshakeServerT* handshake);
-  void sendFrame(const torchcraft::fbs::FrameT* frame);
+  void sendFrame(const flatbuffers::FlatBufferBuilder& builder);
   void sendPlayerLeft(const torchcraft::fbs::PlayerLeftT *pl);
-  void sendEndGame(const torchcraft::fbs::EndGameT *endgame);
+  void sendEndGame(const flatbuffers::FlatBufferBuilder& builder);
```

Comment:
You could add a general function `sendFlatbuffer(flatbuffers::DetachedBuffer&& buffer);`, along with something like

```c++
template<typename T>
sendFlatbufferObject(const T* object) {
  flatbuffers::FlatBufferBuilder fbb;
  auto payload = T::TableType::Pack(fbb, obj);
  auto root = torchcraft::fbs::CreateMessage(
    fbb,
    torchcraft::fbs::AnyTraits<typename T::TableType>::enum_value,
    payload.Union());
  torchcraft::fbs::FinishMessageBuffer(fbb, root);
  sendFlatbuffer(fbb.Release());   
}
```

Then you could get rid of all the sendSomething methods.